python/lsst/daf/base/citizen.cc

```diff
@@ -15,7 +15,14 @@ PYBIND11_PLUGIN(_citizen) {
     cls.def_static("census", (int (*)(int, Citizen::memId)) &Citizen::census);
     /* It is strange to expose cout to Python, so instead make a new wrapper */
     cls.def_static("census", [](Citizen::memId id) { Citizen::census(std::cout, id); });
+    // TODO: pybind11 should logically be return_value_policy::take_ownership (dynamically allocated
+    // C++ object, passed to a local Python variable, never explicitly released), but doing so causes
+    // a segfault ("free(): invalid pointer")
```

Comment:
Following the discussion on Slack, I'll leave a different comment saying that there's no appropriate RVP and that `::copy` is a temporary workaround